main.js

```diff
@@ -27,6 +29,11 @@ class Game {
         }
     }
     imageOnClick(event) {
+        if(!this.interval) {
+            let timer=new Timer();
+            this.interval = timer.startTimer();

```

Comment:
`Game` class should not know how `Timer` is implemented. 
Implementation may vary. `setInterval` is only one approach
I think it's better to keep instance of timer class inside the `Game` class instead.
And do all the manipulations by calling Timer's methods.